deps.edn

```diff
@@ -1,8 +1,8 @@
 {:paths   ["src" "resources" "notebooks"]
  :deps    {org.clojure/clojure {:mvn/version "1.11.1"}}
  :aliases {:dev {:extra-deps {org.clojure/clojure         {:mvn/version "1.11.1"}
-                              techascent/tech.ml.dataset  {:mvn/version "7.011"}
-                              scicloj/tablecloth          {:mvn/version "7.007"
+                              techascent/tech.ml.dataset  {:mvn/version "7.014"}
+                              scicloj/tablecloth          {:mvn/version "7.014"
```

Comment:
worth pointing at bcae624a9f790cac335c0aef27455700ca86c859 ?

Reply:
As discussed on slack: since did not need the full/semi/anti join fixes in TC 7.017-SNAPSHOT agreed OK to stick with v7.014 as current latest stable release.